docs/content/contributing/bicep/bicep-contribution-flow/owner-contribution-flow.md

```diff
@@ -0,0 +1,258 @@
+
+{{< /hint >}}
+
+<br>
+
+---
+
+<br>
+
+### 1. Owner Activities and Responsibilities
+
+<!-- TODO: Rephrase sections if required -->
+Familiarise yourself with the responsibilities as **Module Owner** outlined in [Team Definitions & RACI](https://azure.github.io/Azure-Verified-Modules/specs/shared/team-definitions/#module-owners) and [Module Owner Responsibilities](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/brm-issue-triage/#module-owner-responsibilities) in the [BRM Issue Triage](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/brm-issue-triage/).
+
+1. Create GitHub teams as outlined in [SNFR20](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only) and add respective parent teams:
+
+Segments:
+
+- `@Azure/avm-res-<RP>-<modulename>-module-owners-bicep`
+- `@Azure/avm-res-<RP>-<modulename>-module-contributors-bicep`
+
+Examples:
+
+- `@Azure/avm-res-compute-virtualmachine-module-owners-bicep` and added `@Azure/avm-technical-reviewers-bicep` as parent.
+- `@Azure/avm-res-compute-virtualmachine-module-contributors-bicep` and added `@Azure/avm-module-contributors-bicep` as parent.
+
+If a secondary owner is required, add the secondary owner to the `@Azure/avm-res-<RP>-<modulename>-module-owners-bicep` team.
+
+{{< hint type=info >}}
+
+Once you added the parent teams it needs to be approved by the AVM team to ensure team name and parent team membership is correct.
+
+{{< /hint >}}
+
+2. Add teams to `CODEOWNERS` file as outlined in [SNFR20](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only).
+
+3. Watch Pull Request (PR) activity for your module(s) in the [BRM](https://github.com/Azure/bicep-registry-modules) repository (Bicep Registry Modules repository - where all Bicep AVM modules are published) and ensure that PRs are reviewed and merged in a timely manner as outlined in [SNFR11](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr11---category-contributionsupport---issues-response-times).
+
+{{< hint type=info >}}
+
+Make sure module authors/contributors tested their module in their environment before raising a PR. Also because once a PR is raised a GitHib workflow pipeline is required to be run successfully before the PR can be merged. This is to ensure that the module is working as expected and is compliant with the AVM specifications.
+
+{{< /hint >}}
+
+4. Watch AVM module issue and AVM question/feedback acitvity for your module(s) in the [BRM](https://github.com/Azure/bicep-registry-modules) repository.
+
+5. Ensure that the module(s) you own are compliant with the AVM specifications and are working as expected. Following specifications are to be considered and where `Owner` is mentioned explicitly:
+
+- [SFR1](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-sfr1---category-composition---preview-services)
```

Comment:
It would be nicer to have this in a table format, with a quick description (few words only) of what the given requirement is for - again, just roughly.

Reply:
done